### PR TITLE
add eden

### PR DESCRIPTION
**Pull request by pavanvidem**
* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by bgruening** [comment]
@epruesse no its not, but it also failed with the old one.

File: recipes/eden/1.1/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package:
+  name: eden
+  version: 1.1
```
[the remark this answers is not in the document]
Comment (epruesse) [reply recipes/eden/1.1/meta.yaml]: So it's a case for having `eden/1.1/meta.yaml` and `eden/meta.yaml`, while you are at it?

**Comment by epruesse** [comment]
Try removing the `-static`. It think we don't have `librt.a` and friends (hence the need for libgcc in run depends).

**Comment by epruesse** [comment]
Also, to get `make` to append to a variable you set in the environment, use this:

```
CXXFLAGS?=  # make sure variable exists
CXXFLAGS+= -g -O2
```

**Pull request merged**